### DISCUSSION TITLE
Question about overlapping satins

### DISCUSSION THREAD

**Discussion by sdrose** [inkstitch/inkstitch#1832] [Q&A]
Hello all
I am still fairly new to using Ink/Stitch, but I hve a question about the best approach to overlapping satins.
<img width="1069" alt="Screen Shot 2022-09-24 at 8 45 50 AM" src="https://user-images.githubusercontent.com/13631556/192098726-33fd81ba-9e71-49f0-9a5c-8a20a496ecfb.png">
Here is an example.  I have an overlapping path-to-satin, and I don't think having the satin stitch over itself is the best way.  I used an autoroute because I thought it would break the satin, and, infact, did.  But not at the place where I would like the satin broken. 

I tried to use the path | exclusion (but doesn't work because I have a centerline path with a stroke width.). I thought I could use a pencil line and perform a path | cut line, but while I can cut the path, the cut is at a 90 degree to the centerline and not at the angle i would want the path cut.

I tried to also cut the rails with a path | cut, but that deleted an entire rail.

So, I'd really appreciate some tips and direction on how to get a satin like this to properly stitch out.

I'm using a Brother embroidery machine.  This is an example because what I am doing is digitizing some handwriting - which has non-90 degree angles.

thank you for any and all help/assistance!  I'm hoping the answer ISN'T that what I want can't be done, or is massively time consuming ...

-dave

**Comment by tyrosinase** [comment]
Satins are weird paths because they have the rails, which Inkscape isn't always sure how to deal with for things like Cut, which try to treat the to-be-cut path as though it is a solid fillable object.

What I'd do is take the node tool, and manually put breaks the lines that go under other ones. You'll have to break the paths, then rejoin them with only their own rungs, etc.

![image](https://user-images.githubusercontent.com/9473609/192099827-14b36a82-2649-4153-bf09-5db7d50bc483.png)

Say the horizontal one is the one you want on top; you'd break the vertical one into two paths (actually the bottom one is connected to the horizontal one, so really the blue one would also be green but for the sake of clarity pretend that doesn't happen).

For the ends where they meet the horizontal, you can either extend the short leg over like I did on the red one (in actuality you want it to end underneath the green but I stopped short for, again, clarity) so that the angle of stitches stays consistent, or leave it like the green; the stitch angle will start to swerve after the last rung and end up at the angle between the two endpoints. The red one is better for consistent stitch angle, the blue one is better for consistent stitch length, so it's up to you.

After that, you probably want to order your stitches manually, make sure the path direction is most efficient and so on, so that you can tell it to preserve the order it stitches paths in. It will not, without that, make sure the tails of the cut paths don't ever end up on top of the paths they're going under. You may also end up manually breaking a path that will end up in a continuous-appearing run, just because both of its tails are under itself.

So... probably a little time consuming, but that's the nature of digitizing lettering unfortunately.
